Docs/sphinx_documentation/source/GPU.rst

```diff
@@ -319,8 +319,8 @@ we provide the helper function ``setup_target_for_cuda_compilation()``:
 
 
 
-Enabling HIP support (experimental)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Enabling HIP support
```

Comment:
I think "support" should be capitalized as "Support". This way the titles remain consistent.

Reply:
```suggestion
Enabling HIP Support
```